cmds/record.c

```diff
@@ -1510,14 +1553,11 @@ static void check_binary(struct opts *opts)
 	};
 
 again:
-	pr_dbg3("checking binary %s\n", opts->exename);
+	pr_dbg("checking binary %s\n", opts->exename);
```

Comment:
What about moving this under the `find_in_path()`?

Reply:
If I move this message inside `find_in_path`, then it will not print anything when the user gives the full pathname. The first `is_regular_executable` check will be successful so doesn't call `find_in_path` in this case.

Reply:
Oh, I meant the callsite - as `find_in_path()` can change the exename, it'd be nice to print the final name IMO.

Reply:
I see. I think it's enough to place after the searching the binary as I updated. Thanks.